ambari/src/main/java/io/brooklyn/ambari/hostgroup/AmbariHostGroupImpl.java

```diff
@@ -75,8 +76,12 @@ public void init() {
     public Collection<Entity> resizeByDelta(int delta) {
         Collection<Entity> entities = super.resizeByDelta(delta);
 
-        if (delta != 0) {
+        if (delta > 0) {
```

Comment:
We don't.  I don't think I've done anything to make this worse. As far as ambari is concerned it just won't be able to find the removed servers (at least until the ip is recycled and then it just won't be able to access them).